RpcNep5Tracker/RpcNep5Tracker.cs

```diff
@@ -30,16 +30,22 @@ public class RpcNep5Tracker : Plugin, IPersistencePlugin, IRpcPlugin
         private DataCache<Nep5TransferKey, Nep5Transfer> _transfersReceived;
         private WriteBatch _writeBatch;
         private bool _shouldTrackHistory;
+        private bool _recordNullAddressHistory;
+        private uint _maxResults;
+        private bool _shouldTrackNonStandardMintTokensEvent;
```

Comment:
Does this need to be added to `config.json`?

Reply:
There is no necessity to add it. It will default to false.